developers/integrations/index.md

```diff
@@ -6,25 +6,32 @@ image: og/integrations/home.jpg
 
 Weaviate's integration ecosystem enables developers to build various applications leveraging Weaviate and another technology.
 
-You can find all notebooks and examples on [Weaviate Recipes](https://github.com/weaviate/recipes)!
+All the notebooks and code examples are on [Weaviate Recipes](https://github.com/weaviate/recipes)!
+
+<img
+    src={require('./ecosystem.png').default}
+    alt="alt"
+    style={{ maxWidth: "60%", display: "block", marginLeft: "auto", marginRight: "auto"}}
+/>
+
 
 ## About the Categories
-The ecosystem is broken out by 5 categories:
+The ecosystem is broken into five categories:
 
-* **Cloud Hyperscalers** - 
-* **Cloud Platforms** - 
-* **Data Pipeline** - 
-* **LLM Frameworks** - 
-* **Observability and Evaluation** - 
+* **Cloud Hyperscalers** - Cloud service providers offering large-scale computing and storage
+* **Container Infrastructure** - Platforms and tools for running and scaling containerized applications
+* **Data Pipeline** - Companies offering services and tools for data ingestion and scraping 
+* **LLM Frameworks** - Frameworks used for building generative AI applications
+* **Observability and Evaluation** - Tools to monitor and analyze generative AI workflows
 
 
 
 ## List of Companies
 
 | Company Category | Companies |
 |------------------|-----------|
-| Cloud Hyperscalers | Google, AWS |
-| Cloud Platforms | Replicate |
+| Cloud Hyperscalers | Google, AWS, Azure |
```

Comment:
It would be great for the names to be links to their landing pages. Not necessary by any means, but useful for readers.